src/components/ExpandedBankTransactionRow/ExpandedBankTransactionRow.tsx

```diff
@@ -566,25 +570,19 @@ export const ExpandedBankTransactionRow = forwardRef<SaveHandle, Props>(
                   <div className={`${className}__file-upload`}>
                     <FileInput
                       onUpload={async (file: File) => {
-                        const endpoint = `/v1/businesses/${businessId}/bank-transactions/${bankTransaction.id}/documents`
-                        const formData = new FormData()
-                        formData.append('file', file)
-                        formData.append('documentType', 'RECEIPT') // Assuming the document type is known and static
-
-                        try {
-                          const headers = {
-                            method: 'POST',
-                            headers: {
-                              Authorization: `Bearer ${auth.access_token}`,
-                            },
-                            body: formData,
-                          }
-                          const result = await fetch(apiUrl + endpoint, headers)
-                          const resultJson = await result.json()
-                          await fetchDocuments()
-                        } catch (error) {
-                          console.error('Error uploading file:', error)
-                        }
+                        const uploadDocument =
+                          Layer.uploadBankTransactionDocument(
+                            apiUrl,
+                            auth.access_token,
+                          )
+                        const result = await uploadDocument({
+                          businessId: businessId,
+                          bankTransactionId: bankTransaction.id,
+                          file: file,
+                          documentType: 'RECEIPT',
+                        })
+                        console.log(result)
```

Comment:
Let's drop this one.

Reply:
Good catch, removed both console statements